### PR TITLE
Update creating-dynamic-blocks.md

### PR DESCRIPTION
**Pull request by mrmadhat**
## Description
While posts are loading inside the edit method the `posts` variable is null, attempting to access `post[0]` causes an error 

"TypeError: Cannot read property '0' of null" and the block displays "This block has encountered an error and cannot be previewed." 

checking that posts has a value resolves this.

## How has this been tested?
I have tested these changes in browser.

## Types of changes
Bug fix in creating your first block docs

### CODE REVIEW

**Review by chrisvanpatten — APPROVED** [review]
Thanks for the docs contributions! 📖 💟

File: docs/blocks/creating-dynamic-blocks.md [hunk 1]
```diff
@@ -23,7 +23,10 @@ registerBlockType( 'my-plugin/latest-post', {
 			posts: select( 'core' ).getEntityRecords( 'postType', 'post' )
 		};
 	} )( function( props ) {
-		if ( props.posts && props.posts.length === 0 ) {
+		if ( ! props.posts ) {
+			return "Loading...";
+		}
+		if ( props.posts.length === 0 ) {
```
[the remark this answers is not in the document]
Comment (mrmadhat) [reply docs/blocks/creating-dynamic-blocks.md]: Hey @chrisvanpatten I've added the spacing to both this `if` and the one in your other comment 😃

[the remark this answers is not in the document]
Comment (mrmadhat) [reply docs/blocks/creating-dynamic-blocks.md]: Hey @chrisvanpatten I've added the spacing to both this `if` and the one in your other comment 😃

**Comment by chrisvanpatten** [comment]
Just for a sanity check I want @ajitbohra to take a look too ^__^

**Comment by ajitbohra** [comment]
@mrmadhat thank you :+1:

**Pull request merged**